staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/README.md

```diff
@@ -21,8 +20,9 @@ This plugin provides an integration with Azure Active Directory device flow. If
 
    * Replace the `APISERVER_APPLICATION_ID` with the application ID of `apiserver` application
    * Replace `TENANT_ID` with your tenant ID.
+   * For a list of alternative username claims that are supported by the OIDC issuer check the JSON response at `https://sts.windows.net/TENANT_ID/.well-known/openid-configuration`.
```

Comment:
`sub` is a very cryptic ID, which is not nice to map in a binding.

Reply:
Agreed!  We use upn instead ourselves.